include/zephyr/dt-bindings/clock/bflb_clock_common.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_BFLB_CLOCK_COMMON_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_BFLB_CLOCK_COMMON_H_
+
+#define BFLB_CLKID_CLK_ROOT    0
+#define BFLB_CLKID_CLK_RC32M   1
+#define BFLB_CLKID_CLK_CRYSTAL 2
+#define BFLB_CLKID_CLK_BCLK    3
+
+#define BFLB_FORCE_ROOT_RC32M   10
+#define BFLB_FORCE_ROOT_CRYSTAL 11
+#define BFLB_FORCE_ROOT_PLL     12
+
+#define BFLB_RC32M_FREQUENCY MHZ(32)
```

Comment:
this is error prone as it doesn't include necessary header for the `MHZ()` macro, the parent file need to include this file in a particular order for it to compile